packages/node-cli/src/commands/manageaccount/ManageAccountInputHandler.js

```diff
@@ -48,13 +48,21 @@ const ANSWERS_NAMES = {
 
 module.exports = class ManageAccountInputHandler extends BaseInputHandler {
 	constructor(options) {
-        super(options);
-        this._sdkPath = options.sdkPath;
+		super(options);
+		this._sdkPath = options.sdkPath;
 	}
 
 	async getParameters(params) {
+		if (!this._runInInteractiveMode) {
```

Comment:
I thought that getParameters(params) was never executed if not interactive mode. But seems that is not like this. Shouldn't it be handled by our own "framework" and not execute the getPrameters or do exactly as you do here (just forward the parameters)?

Reply:
Yeah I get your point... maybe... However this was done by another story after this one, I have just done the same as in the rest of the commands to fix the bug.

Any further change about functionality or refactorization has to be done in another story because it's out of the scope of this one.